Makefile

```diff
@@ -7,15 +7,22 @@ CHECK := $(foreach dep,$(DEPS),\
 
 all: test build
 
+build: builddir clean npm frontend sippy sippy-daemon
+
 verify: lint
 
 builddir:
 	mkdir -p sippy-ng/build
 	touch sippy-ng/build/index.html
 
-build: builddir clean npm
+frontend:
 	cd sippy-ng; npm run build
-	go build -mod=vendor .
+
+sippy:
+	go build -mod=vendor ./cmd/sippy/...
```

Comment:
if you add this before the `go build ...` in the `sippy` and `sippy-daemon` rules:

```bash
	mkdir -p sippy-ng/build/
	touch sippy-ng/build/index.h
```

You can eliminate most of lines 5-7 in DEVELOPMENT.md and just mention doing a `make sippy` or `make sippy-daemon`.